src/Model/Search/Filters.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace KSearchClient\Model\Search;
+
+use KSearchClient\HasEnums;
+
+
+/**
+ * The filter names that can be used during a search
+ */
+class Filters
+{
+	use HasEnums;
+
+    const MIME_TYPE = 'properties.mime_type';
```

Comment:
this line is slightly shifted, compared to the other attributes. Was this intended?

Reply:
Not intended, but in this project there is no linting of the code yet